features/main/always_identifier.feature

```diff
@@ -0,0 +1,117 @@
+Feature: AlwaysIdentifier support
+  In order to use a hypermedia API
+  As a client software developer
+  I need to be able to force identifier display of related entities
```

Comment:
I need to be able to force to serialize only the identifier of the related resource